lib/moonshot/default_strategy.rb

```diff
@@ -1,13 +1,12 @@
 module Moonshot
   module ParameterStrategy
-    # Default strategy: grabbing every parameter
-    # from the parameter file.
+    # Default strategy: use parameter defined in the parameter file
     class DefaultStrategy
-      def parameters(_, overrides)
-        overrides.map do |key, _|
+      def parameters(params, _, _)
+        params.map do |key, _|
           {
             parameter_key: key,
-            parameter_value: overrides[key],
+            parameter_value: params[key],
```

Comment:
I think this comment is referring to the line 15 of merge_strategy.rb but is still relevant :)